go/chat/boxer.go

```diff
@@ -219,6 +222,10 @@ func (b *Boxer) boxMessageV1(ctx context.Context, msg chat1.MessagePlaintextV1,
 		}
 	}
 
+	if len(msg.ClientHeader.TlfName) == 0 {
```

Comment:
OK sounds good. I was just suggesting to put in the error message to suggest this might be due to API incompatibility.

I wonder why API incompatibility results in an empty response but not an error though. If (de-)serialization fails in RPC, it should error right?